aten/src/ATen/native/transformers/cuda/library.cpp

```diff
@@ -0,0 +1,26 @@
+#include <ATen/native/transformers/library.h>
+#include <ATen/native/nested/NestedTensorTransformerFunctions.h>
+#include <ATen/native/transformers/transformer.h>
+
+#include <torch/library.h>
+
+namespace at {
+namespace native {
+TORCH_LIBRARY_IMPL(nativetransformers, CUDA, m) {
+  m.impl(
+      "_transform_bias_rescale_qkv_op",
+      TORCH_FN(transform_bias_rescale_qkv_op_cuda));
```

Comment:
huh, is there a reason these aren't in the aten namespace (and then you could have used `native_functions.yaml` for them)

Reply:
deleted!